packages/admin-ui/src/common/routes.ts

```diff
@@ -648,6 +662,8 @@ export const routesData: { [P in keyof Routes]: RouteData } = {
   httpsPortalMappingsRecreate: {},
   httpsPortalExposableServicesGet: {},
   ipfsTest: {},
+  ipfsClientTargetSet: { log: true },
+  ipfsClientTargetGet: { log: true },
```

Comment:
Only log for relevant state changing methods that are not called automatically. If you log on get it's gonna spam the activity monitor